osidb/models.py

```diff
@@ -472,6 +474,22 @@ def __str__(self):
 class FlawManager(ACLMixinManager, TrackingMixinManager):
     """flaw manager"""
 
+    def create(self, jira_token=None, **obj_data):
+        """
+        override create method in order to sync flaw created with Taskman
+        """
+        # imports here to prevent cycles
+        from apps.taskman.service import JiraTaskmanQuerier
+
+        # only creates in database if task were created in Jira
+        with transaction.atomic():
+            flaw = super().create(**obj_data)
+            if not JIRA_TASKMAN_AUTO_SYNC_FLAW or jira_token is None:
+                return flaw
+
+            JiraTaskmanQuerier(token=jira_token).create_or_update_task(flaw)
+            return flaw
```

Comment:
There's a couple of flaws with this approach:

* First, this doesn't cover all the ways that one can create a Flaw, for the sake of simplicity we'll assume that nobody will ever create a Flaw through pure SQL, but they can still do so through `create_flaw` or by using `Flaw`'s constructor and bypass this check.
* This shouldn't prevent the creation of flaws at all, we create flaws outside of the request-response lifecycle all the time (e.g. collectors), we should only block the creation through the API